openedx/core/djangoapps/catalog/models.py

```diff
@@ -1,15 +1,19 @@
 """Models governing integration with the catalog service."""
-from django.utils.translation import ugettext_lazy as _
+from config_models.models import ConfigurationModel
+from django.conf import settings
+from django.contrib.auth import get_user_model
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
-from config_models.models import ConfigurationModel
+from openedx.core.djangoapps.site_configuration import helpers
 
 
 class CatalogIntegration(ConfigurationModel):
     """Manages configuration for connecting to the catalog service and using its API."""
     API_NAME = 'catalog'
     CACHE_KEY = 'catalog.api.data'
 
+    # TODO Replace all usages of this field with a call to get_internal_api_url().
```

Comment:
Consider updating the help text or verbose name to indicate that this field is deprecated. This comment won't be visible to people using the admin.